assig1/src/main/java/ro/utcn/sd/btn/assig1/controller/ConsoleController.java

```diff
@@ -0,0 +1,145 @@
+package ro.utcn.sd.btn.assig1.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import ro.utcn.sd.btn.assig1.exception.QuestionNotFoundException;
+import ro.utcn.sd.btn.assig1.exception.TagNotFoundException;
+import ro.utcn.sd.btn.assig1.exception.UserNotFoundException;
+import ro.utcn.sd.btn.assig1.model.Question;
+import ro.utcn.sd.btn.assig1.model.Tag;
+import ro.utcn.sd.btn.assig1.model.User;
+import ro.utcn.sd.btn.assig1.service.QuestionManagementService;
+import ro.utcn.sd.btn.assig1.service.TagManagementService;
+import ro.utcn.sd.btn.assig1.service.UserManagementService;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Scanner;
+
+@Component
+@RequiredArgsConstructor
+public class ConsoleController implements CommandLineRunner {
+    private final Scanner scanner = new Scanner(System.in);
+    private final InputStreamReader isr=new InputStreamReader(System.in);
+    private final BufferedReader br= new BufferedReader(isr);
+
+    private final QuestionManagementService qManagementService;
+    private final UserManagementService uManagementService;
+    private final TagManagementService tManagementService;
```

Comment:
Naming conventions not the best :) Would have been better to have `questionService` instead of `qManagementService`, `userService` instead of `uManagementService` and so on.